project/BalboaHttp.scala

```diff
@@ -2,19 +2,15 @@ import Dependencies._
 import sbt.Keys._
 import sbt._
 import scoverage.ScoverageSbtPlugin
-//import com.earldouglas.xwp.JettyPlugin
 import org.scalatra.sbt.ScalatraPlugin
 
 object BalboaHttp {
   lazy val settings: Seq[Setting[_]] =
-    //(project in file(".")).enablePlugins(JettyPlugin).settings ++
     BuildSettings.projectSettings ++ Seq(
     libraryDependencies <++= scalaVersion {libraries(_)},
     ScoverageSbtPlugin.ScoverageKeys.coverageMinimum := 0
   ) ++ ScalatraPlugin.scalatraFullSettings
 
-  // balboa-http can not be cross compiled because of the dependency on
```

Comment:
There is another comment in the build.sbt that can be fixed up too. You should only need to remove the part about balboa-http being incompatible with Scala 2.11. Leave the rest. It can be cleaned up, as necessary, when we update the repo to use Scala 2.11.